spec/models/dialog_field_sorted_item_spec.rb

```diff
@@ -1,4 +1,8 @@
 describe DialogFieldSortedItem do
+  before(:each) do
+    allow(dialog_field).to receive(:force_multi_value).and_return(false)
+  end
+
```

Comment:
I don't think this needs to be at the top level. It should be moved into a deeper context within the `#values` describe method, and there needs to be a spec relevant to when it's set to false and when it's set to true.

```ruby
describe "#values" do
  # setup that exists

  context "when the field is dynamic" do
    # setup that exists

    context "when force_multi_value is set to false" do
      # this is where you would do your allow

      context "when the default_value is included in the list of returned values" do
        # this context already exists and you would simply reuse the tests here
      end
    end

    context "when force_multi_value is set to true" do
      # this is where you would do your allow, but return true

      context "when the default_value is included in the list of returned values" do
        # this is the same existing context but now force_multi_value is true and there
        # would be different 'let(:default_value)' values, and different 'it' blocks
      end
    end
  end
end
```

Actually, though, the relevant spec needs to be in the `dialog_field_drop_down_list_spec.rb` because we need to make sure that the private method `default_value_included?` isn't crashing like it was in the bug. So, similar to the contexts I provided above, but there's a few more in the drop down list spec that you will need to integrate.

Regardless, we definitely need a spec that proves the bug is fixed 😄